### PR TITLE
Add automation to chronoskimmer

### LINKED ISSUE #4107: Feature Request: Chronoskimmer Dedication initiative-handling.
Chronoskimmer Dedication has a feature that lets you roll initiative in one of two special ways. Automating this process would significantly ease implementation in a game. Thank you.

### PR DESCRIPTION
**Pull request by KSops**
Resolves #4107

I'm not entirely sure whether I need to follow in the steps of the example for assurance where other modifiers need to be suppressed.
Adds roll options to improve automation. Does not automate the flat check.

### CODE REVIEW

**Comment by stwlam** [comment]
We shouldn't have multiple toggles for statuses that are supposed to be mutually exclusive.

File: packs/data/feats.db/chronoskimmer-dedication.json [hunk 1]
```diff
@@ -24,7 +24,65 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "initiative",
+                "key": "RollOption",
+                "label": "Chronoskimmer Stabilize",
```
Comment (TikaelSol) [inline packs/data/feats.db/chronoskimmer-dedication.json:31]: At the very least we need to localize these, but I'm not keen on having 3 toggles on the sheet for something that can safely live without a toggle like assurance gets.

**Comment by kyles-ep** [comment]
I agree with the point that three toggles isnt an elegant solution, but I'm not sure what a better approach would be given the current set of tools, or what a proposed solution could be for modifying what we have.
I'm open to suggestions.

**Comment by KSops** [comment]
I agree with the point that three toggles isnt an elegant solution, but I'm not sure what a better approach would be given the current set of tools, or what a proposed solution could be for modifying what we have.
I'm open to suggestions.

File: packs/data/feats.db/chronoskimmer-dedication.json [hunk 2]
```diff
@@ -24,7 +24,65 @@
...
+            },
+            {
+                "domain": "initiative",
+                "key": "RollOption",
+                "label": "Chronoskimmer Destabilize (Fail)",
+                "option": "chronoskimmer-destabilize-fail",
+                "toggleable": true,
+                "value": false
+            },
+            {
```
Comment (TikaelSol) [inline packs/data/feats.db/chronoskimmer-dedication.json:52]:
```suggestion
        "rules": [
            {
```
Just get rid of them, roll substitutions maybe make sense to live in the roll dialog.

File: packs/data/feats.db/chronoskimmer-dedication.json [hunk 3]
```diff
@@ -24,7 +24,65 @@
...
+                "effectType": "fortune",
+                "key": "SubstituteRoll",
+                "label": "Chronoskimmer Destabilize (Fail)",
+                "predicate": [
+                    "chronoskimmer-destabilize-fail"
+                ],
+                "required": true,
+                "selector": "initiative",
+                "value": 1
+            }
```
Comment (TikaelSol) [inline packs/data/feats.db/chronoskimmer-dedication.json:84]:
```suggestion
                "label": "PF2E.SpecificRules.Chronoskimmer.StabilizeLabel",
                "predicate": [
                    "chronoskimmer-stabilize"
                ],
                "required": true,
                "selector": "initiative",
                "value": 10
            },
            {
                "effectType": "fortune",
                "key": "SubstituteRoll",
                "label": "PF2E.SpecificRules.Chronoskimmer.DestabilizeLabelSuccess",
                "predicate": [
                    "chronoskimmer-destabilize-success"
                ],
                "required": true,
                "selector": "initiative",
                "value": 19
            },
            {
                "effectType": "fortune",
                "key": "SubstituteRoll",
                "label": "PF2E.SpecificRules.Chronoskimmer.DestabilizeLabelFailure",
                "predicate": [
                    "chronoskimmer-destabilize-fail"
                ],
                "required": true,
                "selector": "initiative",
                "value": 1
            }
```

**Review by TikaelSol — COMMENTED** [review]
Then just add these localizations to re-en.json

**Comment by KSops** [comment]
Thanks for the suggestions! Updated PR.
Will the commits squash or should I force push a squash?

File: static/lang/re-en.json [hunk 4]
```diff
@@ -1050,6 +1050,11 @@
             "CelestialResistance": {
                 "Prompt": "Select a type of energy associated with your bloodline."
             },
+            "Chronoskimmer": {
+                "DestabilizeFailureLabel": "Chronoskimmer Destabilize (Fail)",
+                "DestabilizeSuccessLabel": "Chronoskimmer Destabilize (Success)",
+                "StabilizeLabel": "Chronoskimmer Stabilize"
```
Comment (TikaelSol) [inline static/lang/re-en.json:1056]:
```suggestion
                "DestabilizeFailureLabel": "Destabilize timestream (Failure)",
                "DestabilizeSuccessLabel": "Destabilize timestream (Success)",
                "StabilizeLabel": "Stabilize timestream"
```
These sound better to me I think.

**Comment by TikaelSol** [comment]
Data looks fine, I'll leave it to the code leads for approving the code bits, even if they are minor.

**Comment by TikaelSol** [comment]
Needs rebased

**Comment by KSops** [comment]
Squashed into two separate commits and rebased. I thought it would be good to clearly delineate the data changes from the code changes.

**Pull request merged**